text/0000-pack-pull-policy.md

```diff
@@ -0,0 +1,72 @@
+# Meta
+[meta]: #meta
+- Name: `pack` Image Pull Policy
+- Start Date: 2020-05-21
+- Author(s): @jromero
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+
+Replace the flag `--no-pull` with `--pull-policy <option>` and default to "IfNotPresent"
+
+# Motivation
+[motivation]: #motivation
+
+#### Performance
+
+By not having to check for the latest images on every build we improve performance.
+
+#### Uncommon
+
+The docker CLI, docker compose, and k8s default to similar functionality as described by "IfNotPresent".
+
+# What it is
+[what-it-is]: #what-it-is
+
+A new flag `--pull-policy` that takes the following possible options:
+
+- `if-no-present` - only attempt to pull the images if it's not present
+- `always` - attempt to always pull images
+- `never`- don't attempt to pull images
+
+# How it Works
+[how-it-works]: #how-it-works
+
+<!-- This is the technical portion of the RFC, where you explain the design in sufficient detail.
+
+The section should return to the examples given in the previous section, and explain more fully how the detailed proposal makes those examples work. -->
+
+### `pack build`
+
+// TODO: Detail all the images used during build and ensure that a single flag/policy makes sense for all images or detail exclusions.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+
+# Alternatives
```

Comment:
would it be possible to keep `--no-pull` as a sort of alias for `--pull-policy never`?

Reply:
Is this purely for backwards compatibility cases in which we should note it as such via a deprecation warning and eventually remove it? Or are you thinking of it as shorthand?

Reply:
A vote for deprecated compatibility flag.

Reply:
Added under "Backwards Compatibility"